src/components/ListItem.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <div>
+    <template>
+      <v-container>
+        <v-row>
+          <v-col col="12" md="6">
+            <v-text-field
+              style="margin-bottom:20px;"
+              :value="listItem.text"
+              @change="listItem.text = $event"
+              :outlined="isOutlined"
+              @click="outline"
+              @blur="handleBlur"
+              >{{ listItem.text }}</v-text-field
+            >
+          </v-col>
+          <v-col col="12" md="6">
+            <v-select
+              style="margin-bottom:20px;"
+              :items="allItemsStates"
+              :label="listItem.state"
+              @change = "listItem.state = $event"
+              :outlined="isOutlined"
+              @click="outline"
+              @blur="handleBlur"
+            ></v-select>
+          </v-col>
+        </v-row>
+      </v-container>
+    </template>
+  </div>
+</template>
+
+<script>
+export default {
+  props: {
+    listItem: {
+      type: Object,
+      default: () => ({}),
+      description: 'The list item',
+    },
+  },
+  data: () => ({
+    isOutlined: false,
+  }),
+  methods: {
+    handleBlur(e) {
+      this.isOutlined = false;
+      console.log('blur', e.target);
+    },
+    outline(e) {
+      this.isOutlined = true;
+      console.log('outline', e.target);
+    },
+    changeInputTextField(e) {
```

Comment:
This method name is misleading. The code looks like it updates the value of the listitem text, not the value of the input text. Maybe changing the input text is a side effect of this method, but it's definitely not what it does. There is also the issue of having a single-character variable as an input parameter, same as above.